libs/ledger-live-common/src/families/cosmos/chain/Osmosis.ts

```diff
@@ -15,7 +14,6 @@ class Osmosis extends CosmosBase {
     this.unbondingPeriod = 14;
     this.stakingDocUrl =
       "https://support.ledger.com/hc/en-us/articles/6235986236957-Earn-Osmosis-OSMO-staking-rewards-in-Ledger-Live?docs=true";
-    this.minimalTransactionAmount = new BigNumber(10);
```

Comment:
We don't want any for osmosis ?

Reply:
not relevant any more

Reply:
I removed that because it would use the value from the parent class